sse2neon.h

```diff
@@ -5192,6 +5192,38 @@ FORCE_INLINE __m128d _mm_cmpnge_sd(__m128d a, __m128d b)
 // https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_cmpngt_sd
 #define _mm_cmpngt_sd _mm_cmple_sd
 
+// Compare packed double-precision (64-bit) floating-point elements in a and b
+// to see if neither is NaN, and store the results in dst.
+// https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_cmpord_pd
+FORCE_INLINE __m128d _mm_cmpord_pd(__m128d a, __m128d b)
+{
+#if defined(__aarch64__)
+    // For comparing two quiet NaN, we would get the result that they are not
+    // equal. In orther words, NaN1 != NaN2
```

Comment:
Rewrite as:
```
Compare two quiet NaN would get the result that they are not equal.
In other words, NaN != NaN.
```

Why using `NaN` instead of `NaN1` is because we are comparing the same value in this implementation.